apps/erlang_ls/test/els_test_utils.erl

```diff
@@ -55,6 +55,7 @@ init_per_suite(Config) ->
                            , ?TEST_APP]),
   RootUri = els_uri:uri(RootPath),
   application:load(erlang_ls),
+  application:load(els_framework),
```

Comment:
Do we still need to load `erlang_ls`?

Reply:
No, there is no application config in either els_dap or erlang_ls anymore